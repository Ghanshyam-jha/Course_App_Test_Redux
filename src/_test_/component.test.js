import React from 'react'
import Footer from '../component/footer'
import {create} from 'react-test-renderer'

describe('Testing Component', ()=>{
    test('Testing Footer', () => {
        let tree= create(<Footer/>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
})