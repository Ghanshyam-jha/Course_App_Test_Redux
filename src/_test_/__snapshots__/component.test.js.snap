// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Component Testing Footer 1`] = `
<footer>
  <hr />
  <center>
    <h5>
      Â© Learning Developer.
    </h5>
  </center>
</footer>
`;
